agent.py

```
@@ -431,11 +431,24 @@ async def stream_callback(chunk: str, full: str):
                             # Use the potentially modified full text for downstream processing
                             await self.handle_response_stream(stream_data["full"])
 
+                        async def tokens_callback(text: str, token_count: int):
```

Ruff (0.14.10)

Missing return type annotation for private function `tokens_callback`

Add return type annotation: `None`

(ANN202)